src/DataSets/DataSets.component.js

```diff
@@ -173,10 +181,21 @@ const DataSets = React.createClass({
     openLogs() {
         // Retrieve the logs and save them in this.state.logs, and set
         // this.state.logsObject to a description of their contents.
+        const title = this.tr("logs") + " (" + this.tr("all") + ")";
+        this.setState({
+            logsObject: title,
+            logs: null,
+        });
         getLogs().then(logs => {
-            this.setState({logs: _(logs).orderBy('date', 'desc').value().map(LogEntry)});
+            this.setState({logs: _(logs).orderBy('date', 'desc').value()});
+        });
+    },
+
+    addLogPage() {
+        getLogs([-2]).then(logs => {
```

Comment:
This `-2` is a bit cryptic, where does it come from? Is `addLogPage` used? I see no calls.

Reply:
Yes, sorry, this is really a placeholder for the part that will load more log pages when requested. The `-2` at this moment makes reference to the page previous to the already loaded ones (by default, pages `[-1, 0]` are loaded).

I should at the very least added a comment, if not removed the function completely.